src/app/routes/_broadcast.router.js

```diff
@@ -1,8 +1,33 @@
 import Broadcast from '../models/broadcast.model.js';
+import youtubeHelper from '../helper/youtube.js';
 
 export default (app, router, admin) => {
 
   /**
+   * Repetetly checks for a broadcast record on youtube being published.
+   * Adds link to broadcast if it finds the processed video.
+   *
+   * @param  {Broadcast} broadcast  Model of broadcast without link
+   */
+  async function checkBroadcastLink(broadcast, currentTry = 0) {
+    // Check every 30 minutes, max. 5 times
+    const interval = 1800000;
+    const maxTries = 5;
+    const findBroadcast = await youtubeHelper.findBroadcastURL(broadcast);
+
+    if (findBroadcast.items && findBroadcast.items.length !== 0) {
+      broadcast.videoUrl = 'https://www.youtube.com/watch?v=' + findBroadcast.items[0].id.videoId;
```

Comment:
The link has to be the `https://youtu.be/VIDEOID` in order to work in our implementation.